.github/workflows/android.yml

```diff
@@ -1,9 +1,12 @@
 name: Android CI
 
 on:
-  push:
-    branches: [ beta ]
-  pull_request:
+   workflow_dispatch :
+   schedule :
+    - cron: '0 0 * * *'
+   push:
+      branches: [ beta ]
```

Comment:
There is no beta branch. It should be main. (currently it's master, but I will rename it)

Reply:
okay. I will change it to main.